### PR TITLE
[3.x] fix truncate to delete items in collection

### PR DESCRIPTION
**Pull request by Yahatix**
Make the truncate function only remove documents and not drop the collection

Resolves: #1306

### CODE REVIEW

File: src/Jenssegers/Mongodb/Query/Builder.php [hunk 1]
```diff
@@ -697,16 +697,13 @@ public function from($collection, $as = null)
 
     /**
      * @inheritdoc
+     * @return boolean
      */
-    public function truncate()
+    public function truncate(): bool
```
Comment (Smolevich) [inline src/Jenssegers/Mongodb/Query/Builder.php]: if you added type hints, remove doc blocks please. 
We want to use common style in php classes, so i suggest to delete return type hints and add type hints in refactoring task in future

**Comment by divine** [comment]
This should be pushed to `develop` branch not `master`, since this is a breaking change.

**Comment by Yahatix** [comment]
Sorry this is my first ever merge request and i'm not quite sure how to correct the things you guys have told me. How can i "append" a new commit to this pull request? Or should i make a new pr on the develop branch?

**Comment by Smolevich** [comment]
I think that you left this PR
@divine this is bug and i think that we should fix in release `3.6.3` based on branch `master`

**Comment by divine** [comment]
> Sorry this is my first ever pull request and i'm not quite sure how to correct the things you guys have told me. How can i "append" a new commit to this pull request? Or should i make a new pr on the develop branch?

Just edit your https://github.com/Yahatix/laravel-mongodb and changes will be reflected here.

--
@Smolevich ok, no problem.

**Comment by divine** [comment]
@Smolevich any objections?

**Pull request merged**